apiclient/harvester_api/managers/addons.py

```diff
@@ -0,0 +1,30 @@
+from collections.abc import Mapping
+
+from harvester_api.models.addons import BaseAddonSpec, MonitoringAddonSpec
+from .base import BaseManager, merge_dict
+
+
+class AddonManager(BaseManager):
+    PATH_fmt = "v1/harvester/harvesterhci.io.addons/{uid}"
+    Spec = BaseAddonSpec
+    MonitoringAddonSpec = MonitoringAddonSpec
+
+    def get(self, uid="", *, raw=False):
+        return self._get(self.PATH_fmt.format(uid=uid))
+
+    def update(self, uid, spec, *, raw=False, as_json=True, **kwargs):
+        path = self.PATH_fmt.format(uid=uid)
+        _, data = self.get(uid)
+        if isinstance(spec, self.Spec):
+            spec = spec.to_dict(data)
+        elif isinstance(spec, Mapping) and as_json:
+            spec = merge_dict(spec, data)
+        return self._update(path, spec, raw=raw, as_json=as_json, **kwargs)
+
+    def enable(self, uid, enable=True):
```

Comment:
Can we add `disable` method instead of handling with `enable=False`? It will be similar but more readable & convenient while using it. https://github.com/harvester/tests/pull/1350/files#diff-a398935b69540b5832e6addb8b4e43c3344826e390608865d887b1a98107ca4bR45

Reply:
To implement like this is considered to align the method in `HostManager`:
https://github.com/harvester/tests/blob/7fa14f5251356ce99faa3d993f718e1ea3589031/apiclient/harvester_api/managers/hosts.py#L29

of course we can still add a method `disable` as the shortcut to call `enable(..., False)`, then it will have only 1 argument, is it necessary? :thinking:

Reply:
I think, it will still be convenient for people to look for `disable` method when they are trying to use this.

Reply:
Should be able to resolve this as `disable()` already added.